beacon-chain/blockchain/receive_block.go

```diff
@@ -209,6 +209,7 @@ func (s *Service) ReceiveBlock(ctx context.Context, block interfaces.ReadOnlySig
 
 	timeWithoutDaWait := time.Since(receivedTime) - daWaitedTime
 	chainServiceProcessingTime.Observe(float64(timeWithoutDaWait.Milliseconds()))
+	dataAvabilityWaitedTime.Observe(float64(daWaitedTime.Milliseconds()))
```

Comment:
There are some error returns between this metrics update and where `daWaitedTime` is set. I think we should call `Observe` right after line 124.